src/authorization/utils.ts

```diff
@@ -44,3 +45,44 @@ export const getEpochDifferenceInMS = (
   }
   return parsedFuture - parsedNow;
 };
+
+export const cancelAxiosRequestAndMakeFetch = (
+  config: AxiosRequestConfig,
+  { email, userID }: { email?: string; userID?: string },
```

Comment:
@martinmckenna just curious as to why this param is an object like so? It's not apparent to me the benefit of putting them together like this unless I'm missing something?

Reply:
so when the customer's code identifies the user (either by email or userID), the identification method they chose will determine what's defined here. So it the customer wrote code like this:


```ts
import { initialize } from '@iterable/web-sdk'

const { setEmail } = initialize(...whateverArgs);

setEmail('email@email.com')
```

then here in this function we'll have an email, but not a user ID. They're in an object like this just so we can be explicit when we're consuming them.

I suppose we could just have one argument called `emailOrUserID` but I like the idea of knowing what we're working with and having an explicit name.

The alternative being:

```ts
import { initialize } from '@iterable/web-sdk'

const { setUserID } = initialize(...whateverArgs);

setUserID('1fdws343')
```

Reply:
but to your point, we totally could take them like this:

```ts
export const cancelAxiosRequestAndMakeFetch = (config: AxiosRequestConfig, email?: string, userID?: string) => {}
```

I just prefer if we had one master "identification information" object 🤷‍♂️